_commands/tcpdump.md

```diff
@@ -0,0 +1,101 @@
+---
+---
+
+tcpdump
+-------
+
+tcpdump is a tool that allows you to view a dump of traffic on a network. By default, tcpdump will let you view the dump on your local network.
+
+~~~ bash
+$ tcpdump 
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### Example command
+
+tcpdump -S host <hostname>
+
+~~~ bash
+$ sudo tcpdump -S host www.google.com
```

Comment:
Instead of showing sudo, could you add the steps from here
http://askubuntu.com/questions/530920/tcpdump-permissions-problem so that you
don't have to use sudo with tcpdump?

  

It's generally a bad idea to use sudo with tcpdump or wireshark (or really
anything that doesn't need it). sudo gives you EVERY permission (dangerous),
this method gives you only the few permissions you need for tcpdump to do its
work.

> On Mar 25 2016, at 11:42 am, Matt Terwilliger
&lt;notifications@github.com&gt; wrote:  

>

> In [_commands/tcpdump.md](https://github.com/c4cs/c4cs.github.io/pull/183#di
scussion_r57455925):

>  
>  
>     > +tcpdump is a tool that allows you to view a dump of traffic on a
network. By default, tcpdump will let you view the dump on your local network.

>     > +

>     > +~~~ bash

>     > +$ tcpdump

>     > +~~~

>     > +

>     > +<!--more-->

>     > +

>     > +### Useful Options / Examples

>     > +

>     > +#### Example command

>     > +

>     > +tcpdump -S host <hostname>

>     > +

>     > +~~~ bash

>     > +$ sudo tcpdump -S host www.google.com

>  
>

> Explain why you need to use sudo. Also I don't think having so many lines of
tcpdump makes it any more useful. You can probably cut a bunch out.

>

> —  
You are receiving this because you are subscribed to this thread.  
Reply to this email directly or [view it on GitHub](https://github.com/c4cs/c4
cs.github.io/pull/183/files/a4b10c7915c4b2ee58d2119e79d7195a1156fddc#r57455925
)![](https://github.com/notifications/beacon/AAUt3okJqFl73_cinybKXRgq9HcU8-MWk
s5pxAJngaJpZM4H4iTS.gif)